test/constraints.txt

```diff
@@ -0,0 +1,62 @@
+alabaster==0.7.12
```

Comment:
I recommend managing this with https://pip-tools.rtfd.io. And I'd suggest separating pins for different envs: why do you need pins for testing against three completely different Ansible setups *and* docs? I don't think they should influence each other.

Reply:
Also, consider using sphinx-ansible-theme or Furo. Alabaster is half-dead.